### ISSUE TITLE
xmldom vs xmldom-alpha

### ISSUE THREAD

**Issue by brodybits** [xmldom/xmldom#7]
From TooTallNate/plist.js#100 I finally discovered that they published both xmldom and [`xmldom-alpha`](https://www.npmjs.com/package/xmldom-alpha) from the original master branch. I would now say that [`xmldom-alpha`](https://www.npmjs.com/package/xmldom-alpha) is an unsupported package.

**Comment by timbru31** [comment]
Only @jindw seems to be the owner: `npm owner ls xmldom-alpha`.  
Therefore we can't deprecate the package (I assume). :(

@brodybits, do you know if the npm support could help here?

**Comment by brodybits** [comment]
> Only @jindw seems to be the owner: `npm owner ls xmldom-alpha`.
> Therefore we can't deprecate the package (I assume). :(

unless we request help from npm support, yes

> @brodybits, do you know if the npm support could help here?

Yes. We could send an email to all owners (which would be the email address from `npm owner ls xmldom-alpha`) cc support@npmjs.com with request to publish to that package, according to <https://www.npmjs.com/policies/disputes> ("as respectfully as possible").

I have done this kind of thing before but am reluctant to do this right now for a couple of reasons:

- I have some other pressing priorities right now
- I am actually not convinced that there is much of a pressing need to do this

Whoever uses xmldpm-alpha should really upgrade to use xmldom@latest from npm, which is now supported by us. We *could* do npm deprecate on xmldom-alpha, and publish an update to its README.md on npmjs, or we could just add a statement to our documentation that xmldom-alpha is deprecated and recommend that packages such as plist.js upgrade to xmldom@latest. This is what I implied by my statement in <https://github.com/TooTallNate/plist.js/pull/100#issuecomment-568068097>.

Another side point is that we *could* also ask for rights to publish plist.js updates on npm, according to <https://www.npmjs.com/policies/disputes>. I would love to do this but simply do not have the time right now.

Thanks to @kethinov and @timbru31 for your work to support this!

**Comment by timbru31** [comment]
Personally, I’d just ask the npm support team to deprecate the package and leave the current owner etc. as is.

**Comment by karfau** [comment]
I actually did ask the npm support to deprecate the packages after our first security release, but they never responded with anything useful.

I even went through some repos and wrote some issues/mails to people responsible for packages that used `xmldom-alpha` and some switched, usage is down to 10k/week and there are only 7 dependents left, most of which seem to be unmaintained.

Since we also stated in the changelog that we are not planning to release/maintain xmldom-alpha, I will close this issue.
